stablehlo/dialect/VhloOps.cpp

```diff
@@ -68,10 +77,36 @@ ParseResult parseTensorShape(AsmParser& parser,
   return success();
 }
 
+// Print types in parns: (!vhlo.type, !vhlo.type)
```

Comment:
"parentheses"? I don't think this is a particularly conventional abbreviation.

Reply:
meant parens. Changed to "parentheses"